cpp/daal/include/services/daal_defines.h

```diff
@@ -356,9 +356,10 @@ const int SERIALIZATION_RIDGE_REGRESSION_PARTIAL_RESULT_ID    = 105010;
 const int SERIALIZATION_RIDGE_REGRESSION_TRAINING_RESULT_ID   = 105020;
 const int SERIALIZATION_RIDGE_REGRESSION_PREDICTION_RESULT_ID = 105030;
 
-const int SERIALIZATION_K_NEAREST_NEIGHBOR_MODEL_ID           = 106000;
-const int SERIALIZATION_K_NEAREST_NEIGHBOR_BF_MODEL_ID        = 106001;
-const int SERIALIZATION_K_NEAREST_NEIGHBOR_TRAINING_RESULT_ID = 106010;
+const int SERIALIZATION_K_NEAREST_NEIGHBOR_MODEL_ID             = 106000;
+const int SERIALIZATION_K_NEAREST_NEIGHBOR_BF_MODEL_ID          = 106001;
+const int SERIALIZATION_K_NEAREST_NEIGHBOR_TRAINING_RESULT_ID   = 106010;
+const int SERIALIZATION_K_NEAREST_NEIGHBOR_PREDICTION_RESULT_ID = 106020;
```

Comment:
Does BF requires own IDs to distinguish kd-tree and bf training and prediction results?

Reply:
Brute force kNN ids are added